app/controllers/agreements_controller.rb

```diff
@@ -0,0 +1,47 @@
+class AgreementsController < ApplicationController
+  def index
+    @group = Group.find(params[:group_id])
+    @group_transaction = GroupAgreement.includes(:agreement).where(group_id: params[:group_id]).order(created_at: :desc)
```

Comment:
[OPTIONAL] Following Model-View-Controller (MVC) architectural pattern, we usually don't implement SQL Queries inside the controller, we use the model to implement the SQL queries.

Reply:
Thanks a lot for the review, these SQL queries can be used the same way in each view?

Reply:
> Thanks a lot for the review, these SQL queries can be used the same way in each view?

Just to b sure can move these SQL queries inside each corresponding model ?